eva-server/src/main/java/uk/ac/ebi/eva/server/ws/RegionWSServer.java

```diff
@@ -138,42 +140,30 @@ public QueryResponse getVariantsByRegion(@PathVariable("regionId") String region
             annotationMetadata = new AnnotationMetadata(annotationVepVersion, annotationVepCacheVersion);
         }
 
-        List<VariantWithSamplesAndAnnotation> variantEntities;
+        List<VariantWithSamplesAndAnnotation> variantEntities = Collections.emptyList();
 
         try {
-            variantEntities = service.findByRegionsAndComplexFilters(regions,
-                    filters,
-                    annotationMetadata,
-                    excludeMapped,
-                    pageRequest);
-
-            if (variantEntities == null || variantEntities.isEmpty()) {
+            String[] dbNameParts = species.split("_", -1);
+            Optional<String> asmAcc = taxonomyUtils.getAssemblyAccessionForAssemblyCode(dbNameParts[dbNameParts.length - 1]);
+            if (asmAcc.isPresent()) {
                 List<Region> translatedRegions = regions.stream().map(region -> {
                             String regionContig = region.getChromosome();
-                            String[] dbNameParts = species.split("_", -1);
-                            Optional<String> asmAcc = taxonomyUtils.getAssemblyAccessionForAssemblyCode(dbNameParts[dbNameParts.length - 1]);
-                            if (asmAcc.isPresent()) {
-                                String translatedContig = contigAliasService.translateContigToInsdc(regionContig, asmAcc.get(),
-                                        contigNamingConvention);
-                                if (translatedContig.isEmpty() || translatedContig.equals(regionContig)) {
-                                    return null;
-                                } else {
-                                    return new Region(translatedContig, region.getStart(), region.getEnd());
-                                }
+                            ContigAliasChromosome contigAliasChromosome = contigAliasService.getUniqueInsdcChromosomeByName(regionContig, asmAcc.get(),
```

Comment:
This function should also return the naming convention that was used assuming it `null` to begin with otherwise we won't be able to translate back to whatever it was translated from.